doc/doxygen/src/porting-boards.md

```diff
@@ -0,0 +1,294 @@
+Porting boards                                                 {#porting-boards}
+================
+
+At some point you might need to port a new `BOARD` to `RIOT`, either because
+that specific development board is not yet supported or because you have a
+custom `BOARD` for your project.
+
+If you want to port a `BOARD` to `RIOT` you have two choices: doing it
+inside of `RIOTBASE` or outside. In either case the file structure
+is basically the same and moving from one to another is easy.
+
+This guide details the generic structure you need to add a new `BOARD`
+to `RIOT`, the different files as well as their functionality.
+
+_note_: we assume here that your `CPU` and `CPU_MODEL` is already supported
```

Comment:
```suggestion
_Note_: We assume here that your `CPU` and `CPU_MODEL` is already supported.
```
Should at least begin in uppercase (here and below). What is the reason that the doxygen command @note is not used?